CI Pipeline breaks on building the binary

```
    Finished release [optimized] target(s) in 1m 45s
78
error: /Applications/Xcode_12.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip: unrecognized option: --strip-unneeded
79
Usage: /Applications/Xcode_12.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip [-AnuSXx] [-] [-d filename] [-s filename] [-R filename] [-o output] file [...] 
80
cargo-deb: Command strip failed to launch
81
  because: exit code: 1
82
Error: Process completed with exit code 1.
```

see [the failing step](https://github.com/sassman/t-rec-rs/runs/2259211618?check_suite_focus=true)

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -118,7 +118,7 @@ jobs:
   release:
     name: post / github release
     needs: publish
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: setup | rust

